### ISSUE TITLE
Find or Replace cannot find hex character 0x90 in Extended mode

### ISSUE THREAD

**Issue by M1CR0H4CK3R** [notepad-plus-plus/notepad-plus-plus#7707]
### Description of the Issue
When viewing a document that contains random hex data, using Find in Extended mode to find "\x90" always fails, even if there are actually instances of the hex value in the file being viewed.
### Steps to Reproduce the Issue
1. Open a file consisting of any number greater than 0 of 0x90 hex characters
2. Make sure Encoding is set to UTF-8 (Encoding>UTF-8)
3. Open Find or Replace (Ctrl+F or Ctrl+H)
4. Make sure "Extended (\n, \r, \t, \0, \x...)" is selected
5. Type "\x90" in the search box (without quotes)
6. Click "Find Next"

### Expected Behavior
Notepad++ finds and highlights the next 0x90 character in the file

### Actual Behavior
Error 'Find: can't find the text "\x90"' appears at bottom of Find window

### Debug Information
Not applicable

[Screenshot](https://i.imgur.com/Zj2fBoD.png)

**Comment by M1CR0H4CK3R** [comment]
I just realized it doesn't work on ANY nonprinting characters that aren't control characters. Is that a bug, or intended?

**Comment by M1CR0H4CK3R** [comment]
I figured out why. www.regular-expressions.info states that hexadecimal escapes use the current code page, and I realized that 0x90 ASCII is NOT the same character as 0x90 UTF-8, so that's why it fails. Setting the Encoding to ANSI finds the character, but will only be able to highlight it if "View>Show Symbol>Show All Characters" is enabled. So this is not a bug, but a feature. I will now close this issue.
